.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,108 @@
+name: Build Wheels
+
+on:
+  push:
+    branches: [ main ]
+    tags:
+      - '*'
```

Comment:
So this script is executed when a new release is made. Meanwhile, it is also triggered by `release.yml`. That seems to be a duplicate.